### PR TITLE
pull request for lesson 5

### PR DESCRIPTION
**Pull request by OlyaFed**

### CODE REVIEW

File: src/package5_1/package2/Class3.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package package5_1.package2;
+
+public class Class3 {
+}
```
Comment (Evangard) [inline src/package5_1/package2/Class3.java]: Why is this method empty?

File: src/package5_1/package1/Task7.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+//Tips:
+// - Create two different packages
+// - Create 3 classes. 2 classes should be in the first package, 3rd class - in the second package.
+// - Create in one of the classes some package-private members(fields or methods).
+// - In another two classes create objects of the class with package-private members and try to access its members.
+
+
+public class Task7 {
+    public static void main(String[] args) {
+        new Class2();
```
Comment (Evangard) [inline src/package5_1/package1/Task7.java]: I don't see the demonstration of class 1 and class 3.

File: src/package5_1/Task11.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package package5_1;
+//What is the difference between package-private access and protected access?
```
Comment (Evangard) [inline src/package5_1/Task11.java:2]: I don't see an answer for this question. You can write it in comments.

File: src/package5_1/package2/Class3.java (hunk 1 above)
Comment (OlyaFed) [reply src/package5_1/package2/Class3.java:4]: because we needed to create similar classes in different packages, nothing was said about methods

**Review by Evangard — APPROVED** [review]
You can merge it.

**Pull request merged**